### PR TITLE
Update sponsors with a bunch of logos

### PR DESCRIPTION
**Pull request by bsideslauren**
Add sponsor logos for 2020 that have been shared and re-use sponsor logos from 2019 for those sponsors who have signed but not shared new logos. Spot checked websites for any changes in logo. Add "This Could Be You!" logo to the unsold/unsigned sponsor slots.

### CODE REVIEW

**Comment by rlc4** [comment]
Twilio logo - I'm seeing the BSidesSF logo, it that intentional of now?
BugCrowd - logo link broken, should be sponsors_2020 *and* image file needs to be added
Secure Code Warrior - logo should be in sponsors_2020 not sponsors_2019
corelight - see above
NetFlix - same
Scholarship - not sure what happened here, can you verify the fields in sponsors.yaml for this entry?

(Sorry, I know that's a bunch.)

**Comment by rlc4** [comment]
Just read your comment about re-using 2019 for current sponsors until they give us current logos!  You can ignore a few of my previous comments :)

**Comment by bsideslauren** [comment]
> This is missing the image file for `BsidesSF_2020_SponsorshipCouldBeYou.jpg`. Please add the file to this branch and everything should be good to go.

This was already added in a separate previous commit.

**Comment by bsideslauren** [comment]
> Twilio logo - I'm seeing the BSidesSF logo, it that intentional of now?
> BugCrowd - logo link broken, should be sponsors_2020 _and_ image file needs to be added
> Secure Code Warrior - logo should be in sponsors_2020 not sponsors_2019
> corelight - see above
> NetFlix - same
> Scholarship - not sure what happened here, can you verify the fields in sponsors.yaml for this entry?
> 
> (Sorry, I know that's a bunch.)

Twilio - this was a mix-up between Twilio and the one below. Should be updated now, please confirm
BugCrowd, Secure Code Warrior, corelight, Netflix - all using 2019 logos per my comment
Scholarship - I was missing a  " in the image reference. Should be fixed now.

**Comment by bsideslauren** [comment]
> > This is missing the image file for `BsidesSF_2020_SponsorshipCouldBeYou.jpg`. Please add the file to this branch and everything should be good to go.
> 
> This was already added in a separate previous commit.

Sorry didn't realize this branch was behind the master for this commit. I updated this branch with that commit. Thanks!

File: _data/sponsors.yaml [hunk 1]
```diff
@@ -4,153 +4,153 @@
     - icon: "/images/sponsors_2020/HackerOne.png"
       href: "https://www.hackerone.com/careers"
       name: "HackerOne"
+    - icon: "images/sponsors_2019/twilio.png"
```
Comment (rlc4) [inline _data/sponsors.yaml]: This is missing a / at the front, so it doesn't properly get the image.

File: _data/sponsors.yaml [hunk 2]
```diff
@@ -4,153 +4,153 @@
...
-    - icon: "/images/bsides_logo_full-orangeblack.png"
-      name: "Could Be You!"
-      href: "https://drive.google.com/open?id=1hV45cEqTmy_SYfNPDuZxMkK972-S8oQ7"
 
 - level: Contributing
   class: contrib
   sponsors:
+    - name: "Bugcrowd"
+      href: "https://www.bugcrowd.com/about/careers/"
+      icon: "images/sponsors_2019/bugcrowd.png"
```
Comment (rlc4) [inline _data/sponsors.yaml]: Missing / at the beginning

File: _data/sponsors.yaml [hunk 3]
```diff
@@ -4,153 +4,153 @@
...
-      icon: "/images/bsides_logo_full-orangeblack.png"
-    - name: "Sold! (announcement soon)"
-      href: "https://drive.google.com/open?id=1hV45cEqTmy_SYfNPDuZxMkK972-S8oQ7"
-      icon: "/images/bsides_logo_full-orangeblack.png"
+    - name: "Virtru"
+      href: "https://www.virtru.com/careers/"
+      icon: "/images/sponsors_2020/VirtruLogo-VirtruBlue-RBG.png"
+    - name: "Elastic"
+      href: "https://www.elastic.co/about/careers/"
+      icon: "images/sponsors_2020/elastic-logo-V-full color.jpg"
```
Comment (rlc4) [inline _data/sponsors.yaml]: Missing / ...

File: _data/sponsors.yaml [hunk 4]
```diff
@@ -4,153 +4,153 @@
...
+      icon: "/images/sponsors_2020/BSidesSF_2020_SponsorshipCouldBeYou.jpg"
 
 - level:
   sponsors:
     - type: Workshop
-      name: "Could be you!"
+      name: ""
       href: "https://drive.google.com/open?id=1hV45cEqTmy_SYfNPDuZxMkK972-S8oQ7"
-      icon: "/images/bsides_logo_full-orangeblack.png"
+      icon: "/images/sponsors_2020/BSidesSF_2020_SponsorshipCouldBeYou.jpgg"
```
Comment (rlc4) [inline _data/sponsors.yaml]: Double `g` at the end breaks the image reference

**Comment by rlc4** [comment]
I added a commit to this to fix a few typos.  Please check my work and that everything looks fine.

**Comment by bsideslauren** [comment]
LGTM for the fixes - thanks!

**Pull request merged**